packages/validator/src/validator.ts

```diff
@@ -78,7 +82,24 @@ export class Validator {
 
     const clock = new Clock(config, logger, {genesisTime: Number(genesis.genesisTime)});
     const validatorStore = new ValidatorStore(config, slashingProtection, metrics, signers, genesis);
-    this.indicesService = new IndicesService(logger, api, validatorStore, metrics);
+    const indicesService = new IndicesService(logger, api, validatorStore, metrics);
+    // Do not enable doppelganger when started before genesis or first slot of genesis, because
```

Comment:
Maybe skip the whole first epoch? Instead of first slot? If a validator has duties in a slot (say the first slot) then per the spec, the validator should not have any other duties the reminder of that epoch? so the first epoch should actually be skipped